### DISCUSSION TITLE
出现 this.widgets.set is not a function 报错是什么原因呢

### DISCUSSION THREAD

**Discussion by kylewuyinhai** [antvis/X6#1847] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/50390162/155676490-55df0af4-59a5-4440-b84f-8da67b6434fa.png)

**Comment by NewByVector** [comment, marked as answer]
有复现 demo 吗？

**Reply by kylewuyinhai** [reply]
不好意思，是代码冲突了，已经找到原因了
